shared/js/background/features/fire-button.js

```diff
@@ -0,0 +1,305 @@
+import browser from 'webextension-polyfill'
+import { registerMessageHandler } from '../message-handlers'
+import { getCurrentTab } from '../utils'
+import { getExtensionURL } from '../wrapper'
+import { getDomain, parse } from 'tldts'
+import { sendPixelRequest } from '../pixels'
+
+/**
+ * @typedef {object} BurnConfig
```

Comment:
Could you add some more context to this type comment? In particular the since and origins variables.